include/vcpkg/vcpkgpaths.h

```diff
@@ -39,6 +39,12 @@ namespace vcpkg
         std::vector<ToolsetArchOption> supported_architectures;
```

Comment:
In order to get the reason for the unsupported architecture and link to the error message, how can I modify it here?

Reply:
my understanding is that there is no error message; `supported_architectures` is a list of all of the architectures that are installed, so if an architecture isn't installed it just doesn't show up here.

Reply:
`ToolsetArchOption` should have the corresponding component value for external judgment, but because the toolset is a string, it is difficult to correspond to the error message.